addons/website/static/tests/tours/edit_link_popover.js

```diff
@@ -167,6 +167,13 @@ tour.register('edit_link_popover', {
     {
         content: "Click 'Home' link in footer",
         trigger: 'footer a[href="/"]',
+        run: function (actions) {
+            actions.click();
+            const sel = document.getSelection();
+            const el = this.$anchor[0];
+            sel.collapse(el, 0);
+            el.focus();
+        },
```

Comment:
I'm not sure I understand the change in the tour here. Why is it needed when it worked before? Or did it not work properly before? What changed? It looks like before, the popover was displayed just by clicking, while now it requires to trigger `selectionchange` manually? Isn't that a regression of sorts? Unless it should not have opened in that case in the first place? But then it would be kind of surprising considering the next step in the tour specifically tests that... ^^' In short: I'm very confused :p

Reply:
With my change I close the link popover whenever the selection moves out of link. However, during the tour, when clicking on `footer a[href="/"]`, the selection is at node above it(see: 
![image](https://github.com/user-attachments/assets/cd566084-037d-44cf-b70e-d96d2fb619eb)
) which is outside the link causing the popover to close. This just only occurs in tour and I can't reproduce it myself. So the best thing I could do is force the selection to link avoiding the link popover to close.

Reply:
Then I suggest maybe adding a comment explaining that it only happens in tours?

Reply:
@dmo-odoo I just rebased it and the same tour is failing now. But what is confusing me here is [this step](https://github.com/odoo/odoo/pull/171469/files#diff-01d332f2cb28d360c43ef5ffaa293aed9f6335756a5081054f341c18f505a558R198) says double click should not open popover. But the very [next step](https://github.com/odoo/odoo/pull/171469/files#diff-01d332f2cb28d360c43ef5ffaa293aed9f6335756a5081054f341c18f505a558R217) says 'Ensure popover is opened on double click'. Maybe I am not sure what is need here on a functional point of view. :man_shrugging: 

cc. @lebl-odoo

Reply:
The expected behavior is the following:
1. one click -> open the popover
2. two clicks -> select text

For website, when you double click, the focus is on the link edition (sidebar)